c2corg_ui/static/menu-discourse/menu.html

```diff
@@ -0,0 +1,117 @@
+## HEAD
+<link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet">
+
+## En-tete
+<aside id="sidemenu">
+  <div class="page-menu">
+    <a href="//www.demov6.camptocamp.org">
```

Comment:
Can't we avoid hard coded URLs?
It means those links will have to be hand-changed when setting the menu in the prod forum.
Or at least to have a single place ("baseURL"?)?